extensions/objects/protractor.js

```diff
@@ -21,6 +21,19 @@
 // implement a setValue() function to which a single argument can be sent
 // that will completely determine the object.
 
+var AlgebraicExpressionEditor = function(elem) {
+  return {
+    setValue: async function(text) {
+      await elem.click();
+      var algebraicInputElem = element(by.css('.guppy_active'));
+      var present = await algebraicInputElem.isPresent();
```

Comment:
What if it hasn't loaded yet? A `waitFor` might be helpful to make sure the element has loaded before we interact with it

Reply:
Done.